openfpga_flow/scripts/run_fpga_task.py

```diff
@@ -166,8 +166,14 @@ def generate_each_task_actions(taskname):
     """
 
     # Check if task directory exists and consistent
-    curr_task_dir = os.path.join(gc["task_dir"], *(taskname))
-    if not os.path.isdir(curr_task_dir):
+    local_tasks = os.path.join(*(taskname))
```

Comment:
Suggest some comments about the ``local_task`` and ``repo_task`` so that it is easier to maintain these codes. It seems that the local task has a higher priority. Should this lead to an update in the documentation?

Reply:
I will add that in documentation, and while running the task the script print task directory irrespective of the debug flag.
That will help user to see which task is running

Reply:
> O.K. In case we all forget about this, can you comment on the issue #151 ?

Reply:
The deadcode is already removed, the documentation is underway. I will make pull for some of them today .